server.py

```diff
@@ -63,5 +70,34 @@ def check_errors(request):
     return 'No Errors'
 
 
+def resolve_uri(uri):
+    if os.path.isfile(uri):
+        file_content = read_file(uri)
```

Comment:
Just as a question, what happens if the file is not readable?  Maybe a permissions error or something?